### PR TITLE
added generateSeedPhraseWithEntropy

### PR DESCRIPTION
**Pull request by mattlockyer**
optional feature to create seedPhrase with entropy from other key entropy providers (e.g. Torus)

### CODE REVIEW

**Comment by mattlockyer** [comment]
Cannot determine if only using first 32 hex bytes of generated sha512 from nacl will be an issue?

bip39-light only accepts hex entropy strings up to length 32 :thinking:

**Comment by mattlockyer** [comment]
@vgrichina looks like API key is invalid on CI tests

File: index.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ const generateSeedPhrase = () => {
     return parseSeedPhrase(bip39.generateMnemonic())
 }
 
+const generateSeedPhraseWithEntropy = (entropy) => {
```
Comment (vgrichina) [inline index.js]: @mattlockyer lets just make it optional parameter to generateSeedPhrase

Comment (mattlockyer) [reply index.js]: agreed

**Comment by vgrichina** [comment]
> @vgrichina looks like API key is invalid on CI tests

@chefsale may you look into what needs to be fixed with fossa on CI?

**Pull request merged**